### DISCUSSION TITLE
How does submariner do coordination between the controllers in the two clusters?

### DISCUSSION THREAD

**Discussion by kapilagrawal95** [submariner-io/submariner#2727] [General]
Hi,

I have a fundamental question about submariner. Does submariner's broker synchronizes the etcd of each cluster as well? For example, if I have a deployment which I would like to ensure are spread uniformly across both the clusters? How will the coordination between the controllers be in such a situation?

Thanks in advance!

**Comment by tpantelis** [comment]
The broker is just a namespace to which participating clusters read and write CRs to share them between the clusters.

**Reply by kapilagrawal95** [reply]
So does it mean that a global view (etcd) is not available to the kubernetes cluster? And therefore, the use-case I mentioned of spreading replicas across clusters is not supported by submariner currently?

**Reply by tpantelis** [reply]
I assume you're asking if submariner shares the entire K8s DB across clusters. No - it only shares CRs that are necessary for submariner to function properly. Anything else is out-of-scope for the purpose of submariner.

**Reply by sridhargaddam** [reply]
Adding to what @tpantelis mentioned, you can take a look at the Architecture page to understand how Submariner uses the Broker - https://submariner.io/getting-started/architecture/broker/
